src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,41 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static int MAX_ARRAY_VALUE = 10;
```

Comment:
```suggestion
    private static int MAX_ARRAY_CAPACITY = 10;
```
як правило, value ми називаємо щось що зберігаємо(в даному випадку в storage) а це число показує скільки ми можемо сюди покласти